### ISSUE TITLE
Bubbles fading in/out on drag

### ISSUE THREAD

**Issue by jheeffer** [Gapminder/vizabi#1553]
Bubbles are fading in and out on play now, awesome! But same behaviour on drag would be good too right?

**Comment by angieskazka** [comment]
no, as per Ola's requirements all dragging animations should be immediate to avoid delays in user feedback

**Comment by jheeffer** [comment]
In gapminder world it does work like this and current way makes one-year data (like tsunami deaths 2004) no show up at all while dragging.
I don't understand Ola's requirement anyway or how it pertains to this? What user feedback is delayed when fading in/out? It is more, better feedback imo?

**Comment by angieskazka** [comment]
let's reopen and ask Ola
